src/compiler/compiler.cc

```diff
@@ -131,10 +131,6 @@ class Pipeline {
   virtual ast::Unit* parse(Source* source);
   virtual void setup_lsp_selection_handler();
 
-  // Gives the Pipeline the opportunity to change the program once it was
```

Comment:
Maybe merge from main to make this easier to review?

Reply:
done.